index.html

```diff
@@ -105,44 +105,47 @@ <h4>CSS</h4>
                     <span>web development language</span>
                     <h4>CSS</h4>
                 </div>
+                <div class="rating"></div>
+
+
             </div>
 
-            <div class="card">
-                <div class="card-img">
-                    <img src="assets/images/Logos/django.jpeg" alt="image">
+            <div class="card ">
+                <div class="card-img ">
+                    <img src="assets/images/Logos/django.jpeg " alt="image ">
                 </div>
 
-                <div class="card-text">
+                <div class="card-text ">
                     <span>web development language</span>
                     <h4>CSS</h4>
                 </div>
             </div>
-            <div class="card">
-                <div class="card-img">
-                    <img src="assets/images/Logos/devops.png" alt="image">
+            <div class="card ">
+                <div class="card-img ">
+                    <img src="assets/images/Logos/devops.png " alt="image ">
```

Comment:
most of the changes in  this file are adding space to end of the string, is that intentional?

Reply:
No, not intentional
I think this is caused by some extensions in vs code, I will correct them